### ISSUE TITLE
Creepers Explosion Savage & Ravage

### ISSUE THREAD

**Issue by MrEcko19** [spoorn/sodium-forge#21]
### Expected Behavior
The Abnormal Mod Savage & Ravage have and option to disable Creepers destroying blocks in explosions and spawn creeperlings, a new mob.

### Actual Behavior
This is actually not happening. Explosions destroy blocks and creeperlings do not spawn.

### Reproduction Steps
1. Creeper explode
2. Break blocks
3. Don't spawn Creeperlings

### Attachments
Not neccesary

### System Information
- Java Version: 8
- CPU: i7-6700HQ 2,60Ghz
- GPU: Nvidia GTX960m

**Comment by KKotik** [comment]
Probably it's not a bug. By default mod not changing their behavior anymore, you need to turn this on in the config.

**Comment by MrEcko19** [comment]
Hi, I checked it before posting the issue. and also check what mod is creating the bug because it can be anyone that have to be with creepers. I was surprise that Sodium Reforged is changing something about creepers in the code, but this feature seems working for me without Sodium installed, and stop working and creating holes whenever I load Minecraft with it.

**Comment by KKotik** [comment]
Still working fine for me.

**Comment by MegaHakkero** [comment]
This also happens with the CreeperFix mod, which should prevent creepers from damaging the terrain. With both Sodium Reforged and CreeperFix installed, creepers will do environmental damage.

**Comment by The-Bernician-Lamb** [comment]
I can confirm that I've recently noticed this issue as well. It only seems to happen when Sodium is installed. 

In fact I've noticed that this issue doesn't just occur with savage and ravage but basically every mod that changes creepers in some way. I've been using Majruszs progressive difficulty lately: (https://www.curseforge.com/minecraft/mc-mods/majruszs-progressive-difficulty) and sodium also interferes with the creeper features in that mod as well.
